test/src/rabbit_mqtt_test_util.erl

```diff
@@ -0,0 +1,59 @@
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at http://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ.
+%%
+%% The Initial Developer of the Original Code is GoPivotal, Inc.
+%% Copyright (c) 2007-2015 Pivotal Software, Inc.  All rights reserved.
+%%
+
+-module(rabbit_mqtt_test_util).
+
+-include_lib("eunit/include/eunit.hrl").
+
+%%--------------------------------------------------------------------
+%% coerce exchange and vhost to binaries
+%%--------------------------------------------------------------------
+
+%% coerce in case of string
+coerce_exchange_test() ->
+    <<"amq.direct">> = rabbit_mqtt_util:coerce(exchange, "amq.direct").
```

Comment:
Your tests are similar [to these](https://github.com/rabbitmq/rabbitmq-server/commit/9f4ce81d28278f14bc60754640769d49565a6060). To use `application:get_env/2` the app needs to be loaded, but not necessarily running. See [application:load/1](http://erldocs.com/current/kernel/application.html?i=15&search=application:#load/1).